### PR TITLE
Feature/dapp improve display and input components

### PR DESCRIPTION
**Pull request by weilbith**
This PR got split out of  a bigger PR that will come afterwards. It basically includes a couple a smaller improvements which will be used later.

### CODE REVIEW

**Comment by weilbith** [comment]
No CHANGELOG entry necessary for this. No new feature or bugfix for the user.

File: raiden-dapp/src/locales/en.json
Comment (taleldayekh) [inline raiden-dapp/src/locales/en.json:266]: I think I commented on this in the last PR, so I guess it will be changed here once you merge that one.

File: raiden-dapp/tests/unit/components/address-display.spec.ts
Comment (taleldayekh) [inline raiden-dapp/tests/unit/components/address-display.spec.ts:1]: I've seen this a multiple times. Is it so that the identicon is **not** needed and imported anymore or did you just rename the `$identicon` to `$t`?

File: raiden-dapp/tests/unit/components/address-display.spec.ts
Comment (taleldayekh) [inline raiden-dapp/tests/unit/components/address-display.spec.ts:23]: Ah, I guess the identicon is not needed. :)

File: raiden-dapp/tests/unit/components/address-input.spec.ts
Comment (taleldayekh) [inline raiden-dapp/tests/unit/components/address-input.spec.ts:262]: Change "custo" -> "custom".

**Review by taleldayekh — APPROVED** [review]
LGTM 👍🏼

File: raiden-dapp/src/locales/en.json
Comment (weilbith) [reply raiden-dapp/src/locales/en.json:266]: You refer to [this comment](https://github.com/raiden-network/light-client/pull/2921#discussion_r701241354). No, this PR becomes before the other one. So I'll do it here.

File: raiden-dapp/tests/unit/components/address-display.spec.ts
Comment (weilbith) [reply raiden-dapp/tests/unit/components/address-display.spec.ts:1]: The identicon is only necessary for a very few components where it is actually used. While the translation function is basically used everywhere.

File: raiden-dapp/tests/unit/components/address-input.spec.ts
Comment (weilbith) [reply raiden-dapp/tests/unit/components/address-input.spec.ts]: ✔️

File: raiden-dapp/src/locales/en.json
Comment (weilbith) [reply raiden-dapp/src/locales/en.json]: ✔️

**Comment by weilbith** [comment]
All feedback addressed. I'll will squash the fixup commits now.

**Comment by weilbith** [comment]
I'll wait for the CI to confirm and then I'll merge. Afterwards I'll rebase #2921 on top of it.

**Pull request merged**
